src/main/scala/akka/persistence/cassandra/query/journal/CassandraReadStatements.scala

```diff
@@ -0,0 +1,29 @@
+package akka.persistence.cassandra.query.journal
+
+trait CassandraReadStatements {
```

Comment:
Again keeping the read and write side separated as they both have very different needs. This will be removed or largely changed when the index tables are introduced anyway.